tests/test_df2gspread.py

```diff
@@ -310,6 +310,131 @@ def test_df2gspread_start_cell(user_credentials_not_available):
     file_id = get_file_id(credentials, filepath)
     delete_file(credentials, file_id)
 
+def test_df2gspread_df_size(user_credentials_not_available):
+    if user_credentials_not_available:
+        pytest.xfail(reason='Credentials')
+
+    import string
+    import random
+    import numpy as np
+    import pandas as pd
+    import gspread
+    from pandas.util.testing import assert_frame_equal, assert_equal
+
+    from df2gspread import df2gspread as d2g
+    from df2gspread import gspread2df as g2d
+    from df2gspread.utils import get_credentials
+    from df2gspread.gfiles import get_file_id, delete_file, get_worksheet
+
+    filepath = '/df2gspread_tests/' + ''.join(
+        random.choice(string.ascii_uppercase + string.digits)
+        for _ in range(10))
+    credentials = get_credentials()
+    gc = gspread.authorize(credentials)
+    gfile_id = get_file_id(credentials, filepath, write_access=True)
+
+    df_upload_a = pd.DataFrame(
+        {0: ['1', '2', 'x', '4']},
+        index=[0, 1, 2, 3])
+    df_upload_a = df_upload_a.fillna('0')
```

Comment:
I added this method to fill empty dataframe with zeros.. for example [here](https://github.com/jasonqng/df2gspread/blob/f8e5e85388b990389f037d13386c96e22d4fa809/tests/test_df2gspread.py#L209). In case of filled df it do nothing